girder/cumulus/server/volume.py

```diff
@@ -275,11 +263,14 @@ def attach(self, volume, cluster, params):
             raise RestException('Master instance is not running!',
                                 400)
 
+        cluster = self.model('cluster', 'cumulus').filter(
+            cluster, getCurrentUser(), passphrase=False)
```

Comment:
Sorry,  i get confused by girder's model semantics sometimes,  don't we pass in cluster to the attach() function already?